pkg/common/parameters.go

```diff
@@ -208,6 +218,15 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 				return p, fmt.Errorf("parameters contain invalid value for %s parameter: %w", ParameterKeyStoragePools, err)
 			}
 			p.StoragePools = storagePools
+		case ParameterKeyResourceTags:
```

Comment:
Same logic is used here and below in `ExtractAndDefaultSnapshotParameters`. Do you anticipate this logic diverging in the future? If not, consider refactoring to a common method